src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -200,4 +200,29 @@ class Eureka extends Common {
       logger.error("Not able to assign ${resourceIds} to ${roleId}")
     }
   }
+
+  void enableApplicationForTenant(String tenantId, List applications) {
+
+    Map<String, String> headers = getHttpHeaders(masterTenant)
+    headers['x-okapi-token'] = headers['Authorization'].replace('Bearer ', '')
+
+    Map body = [
+      tenantId    : tenantId,
+      applications: applications
+    ]
+
+    String url = "https://${okapiDomain}/entitlements?ignoreErrors=false&purgeOnRollback=true"
```

Comment:
I suppose we should use 'eurekaTenant.kongUrl' attribute here instead of 'okapiDomain'

Reply:
okapiDomain in current implementation is kong URL. Name will be change is near future